### ISSUE TITLE
raw file url issue

### ISSUE THREAD

**Issue by prajwal0201** [darshilparmar/tokyo-olympic-azure-data-engineering-project#8]
whenever i m  clicking on raw, it gets downloaded due to which url of raw data cannot be seen.

**Comment by himanshu8795** [comment]
yes i am also having the same issue , whenever i m clicking on raw, it gets downloaded due to which url of raw data cannot be seen.

**Comment by adityak3108** [comment]
> yes i am also having the same issue , whenever i m clicking on raw, it gets downloaded due to which url of raw data cannot be seen.

right click on the raw button and then select the option copy url. and use that url for pasting futher

**Comment by DivyeMaggo** [comment]
If the file gets downloaded, press 'ctrl+j' , the file name will appear in the recently downloaded items. Right click on the file and choose 'Copy Downloaded Link'.
To avoid doing the same process again and again, just change the last parameter (file name) from the download link while creating other copy data functionality.

**Comment by mak343** [comment]
Hi All , I had similar issue and wasted few days to resolve this issue ( since I learning AZure data factory ) Solution is we need configure Datafactory account to Github respo. please watch https://www.youtube.com/watch?v=rU1ejUBBDPs.

**Comment by Ruturajjadhav35** [comment]
does anyone have a better approach for the same ?

**Comment by NITYASHETTY7** [comment]
> If the file gets downloaded, press 'ctrl+j' , the file name will appear in the recently downloaded items. Right click on the file and choose 'Copy Downloaded Link'. To avoid doing the same process again and again, just change the last parameter (file name) from the download link while creating other copy data functionality.

  Hey this one works,thanks!
Ig it's the problem with that file itself cauz EntriesGender and Medals you get direct raw link.
